styles/mixins/button.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function () {
+  return function (mixin, bgColor, color) {
+    return {
+      'background-color': bgColor,
+      color,
+      display: 'inline-block',
+      'line-height': 1
```

Comment:
yeah, that works for me.  we probably also need a reset -webkit-appearance too, right?